### PR TITLE
torch.flip issue is fixed in SynapseAI 1.16, so remove the WA in opti…

### PR DESCRIPTION
**Pull request by sywangyi**
…mum-habana,

verified in text-to-speech generation of facebook/mms-tts-eng.

# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: optimum/habana/transformers/models/__init__.py [hunk 1]
```diff
@@ -195,7 +195,6 @@
 from .vit import gaudi_vit_self_attention_forward
 from .vits import (
     gaudi_unconstrained_rational_quadratic_spline,
```
Comment (yafshar) [inline optimum/habana/transformers/models/__init__.py:197]:
```suggestion
from .vits import gaudi_unconstrained_rational_quadratic_spline
```

File: optimum/habana/transformers/models/vits/__init__.py [hunk 2]
```diff
@@ -1,4 +1,3 @@
 from .modeling_vits import (
     gaudi_unconstrained_rational_quadratic_spline,
```
Comment (yafshar) [inline optimum/habana/transformers/models/vits/__init__.py:2]:
```suggestion
from .modeling_vits import gaudi_unconstrained_rational_quadratic_spline
```

File: optimum/habana/transformers/models/vits/__init__.py [hunk 3]
```diff
@@ -1,4 +1,3 @@
...
-    gaudi_VitsResidualCouplingLayer_forward,
 )
```
Comment (yafshar) [inline optimum/habana/transformers/models/vits/__init__.py:3]:
```suggestion
```

**Comment by yafshar** [comment]
LGTM!

@regisss Could you take a look at this. I also tested this PR.

**Review by regisss — APPROVED** [review]
LGTM!

**Pull request merged**